scripts/gb_model/compose_network.py

```diff
@@ -603,8 +920,17 @@ def compose_network(
     demands = {
         k.replace("demand_", ""): v
         for k, v in snakemake.input.items()
-        if k.startswith("demand_")
+        if k.startswith("demand_") and k != "demand_fes_transport"
```

Comment:
prefer that this is dealt with when passing demands as inputs to the rule.

Reply:
I see this is how we are avoiding double counting (as I worried about at the rule level), but ignoring `fes_transport` as a demand. If we move this to the rule-level (in the `demands` function) then I think it would be a clearer. It would require you to pass the transport data you _do_ need more explicitly.